/**:
  ros__parameters:
    robot_link: "robot"
    sensor_link: "laser"
    map_link: "odom"
    opt_path_frame_: "odom"
    opt_pose_frame_: "odom"
    use_sim_time: true

slam:
  sensor_offset_node:
    ros__parameters:
      x: 0.421
      y: -0.327
      z: 0.044
      qx: 0.92396
      qy: -0.3825
      qz: 0.0
      qw: 0.0

  slam_demo_node:
    ros__parameters:
      sub_wheel_odom_topic_: "/robot/throttle/odom"
      sub_laser_topic_: "/robot/front_laser/scan"

  rviz2:
    ros__parameters:
      tf_buffer_cache_time_ms: 10000